integration/setup/setup_compat.go

```diff
@@ -43,6 +43,10 @@ type SetupCompatOpts struct {
 	// Data providers.
 	Providers []shareddata.Provider
 
+	// If true, a non-existed collection will be added to the list of collections.
+	// This is useful to test the behavior when a collection is not found.
```

Comment:
When it is _not_ useful?

If it is always useful, why make it configurable?

Reply:
I'm glad you like it! Unfortunately, I can't enable it for all the tests right now, as some tests will fail (and the scope of this task will be increased). So, I created a separate issue: https://github.com/FerretDB/FerretDB/issues/1545